packages/theme/src/components/number-input.ts

```diff
@@ -32,19 +33,25 @@ const baseStyle = (props: Record<string, any>) => ({
   stepper: baseStyleStepper(props),
 })
 
-function getSize(size: "sm" | "md" | "lg") {
-  const sizeStyle = Input.sizes?.[size]
+function getSize(size: "xs" | "sm" | "md" | "lg") {
+  const sizeStyle = Input.sizes[size]
 
   const radius = {
     lg: "md",
     md: "md",
     sm: "sm",
+    xs: "sm",
   }
 
+  const resolvedFontSize = Typography.fontSizes[sizeStyle.field.fontSize]
```

Comment:
Is this fine to do? I am trying to get rid of that `size === "lg" ? "14px" : "10px"` in stepper fontSize.